scripts/al1/configure-docker-storage-setup.sh

```diff
@@ -22,3 +22,5 @@ LV_ERROR_WHEN_FULL=yes
 EXTRA_DOCKER_STORAGE_OPTIONS="--storage-opt dm.fs=ext4 --storage-opt dm.use_deferred_deletion=true"
 EOF
 sudo mv /tmp/docker-storage-setup /etc/sysconfig/docker-storage-setup
+
+sudo /usr/bin/docker-storage-setup
```

Comment:
It's probably worth running this as a cloud-init script rather than part of the AMI build so that changes to the configuration (through modifications to the block device mappings during instance launch or settings written in user-data) can be properly reflected on the instance.

In the previous AMI recipes, we wrote out a file called `/etc/cloud/cloud.cfg.d/90_ecs.cfg` with this contents:

```
#cloud-config
system_info:
  default_user:
    groups: [ "wheel", "docker" ]
bootcmd:
 - [ cloud-init-per, instance, docker_storage_setup, /usr/bin/docker-storage-setup ]
```

Reply:
we are still writing that file, but for some reason I am not seeing it run on boot, I will look into why
https://github.com/aws/amazon-ecs-ami/blob/main/files/al1/90_ecs.cfg

Reply:
I tested without running this command during the AMI build and it turns out the boot-cmd is working as expected, I guess the issue previously was just that it would fail because of the incorrect block device mapping of the AMI.